features/ccx-upgrades-inference/perform_inference.feature

```diff
@@ -1,26 +1,26 @@
 Feature: Upgrade Risks Prediction inference - test well known values
 
   Scenario: Check Inference Service response with no body is sent in the request
-    Given The CCX Inference Service is running
-     When I request the upgrade-risks-prediction endpoint
+    Given The CCX Inference Service is running in port 8000
```

Comment:
"on port" perhaps?

Reply:
Ouch! I always doubt if using "in" or "on" for ports. I'll fix it